packages/web/src/components/spa/HackAUBG/JourneySection/JourneyDesktop/JourneyDesktop.jsx

```diff
@@ -34,14 +36,21 @@ export const JourneyDesktop = () => {
                             STEP 3:<br></br> Get Hackathoning
                         </h2>
                         <p>
-                            Get together with some fellow enthusiasts and
-                            register in the form below! Teams must be between 3
-                            and 6 people. However, if you have less, the Hub
-                            team will help you link up with other hackers
-                            waiting to participate. If the number of registered
-                            teams becomes larger than 10, your team will be
-                            waitlisted until the last week before the
-                            competition.
+                            HackAUBG 5.0 starts on Friday, March 31st. It will
+                            be fully on-ground, taking place in the American
+                            University in Bulgaria’s Sports Hall in the America
+                            for Bulgaria Building. If you don’t reside in
```

Comment:
Change/rephrase the "American University in Bulgaria’s Sports Hall in the America for Bulgaria Building" because it seems too confusing and difficult to read.